libtransmission/peer-io.cc

```diff
@@ -140,6 +142,9 @@ std::shared_ptr<tr_peerIo> tr_peerIo::new_outgoing(
                   auto* const sock = utp_create_socket(session->utp_context);
                   utp_set_userdata(sock, peer_io.get());
                   peer_io->set_socket(tr_peer_socket{ socket_address, sock });
+                  tr_logAddTraceIo(
```

Comment:
Maybe factor out into a function instead of repeating 3 times? Or at least reduce to 2 by moving this logging out of the lambdas? Also looks like it will now potentially log multiple times for the same io in case `utp_connect()` fails, which could be misleading.

Reply:
e5024da